modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js

```diff
@@ -104,6 +104,12 @@
 				}
 			}
 
+			function enableFormButtons(inputs) {
+				Util._submitLocked = null;
```

Comment:
Where is value being initialized at? I see that `_submitLocked` is used in a few places, and I think we need to make sure they are all reference the same value, otherwise we might run into issues.

Reply:
That one was confusing to me, when doing `Liferay.Util._submitLocked` in the console, it returns `false`.

I see it's being set in `js-state-web` and in a couple other places inside `js-web` and `js-aui-web`, but I didn't see that it's being set in the `util.js` file among all other global declarations.

Reply:
We just need to make sure all cases read the value from the same place, otherwise we might ended up with inconsistent values when calling this util vs the global util vs the one from aui-web

Reply:
Why do you think this PR changes anything regarding `Util._submitLocked`? @bryceosterhaus 

Both the deprecated and the new one set its value to `null`. This only happens when the function is invoked, because its inside of the functions scope. I don't see how this could cause any new issues.

Reply:
Oh I see where I am confused at here. I didn't realize you were just inlining this function to where it is used so that it could be deprecated in aui-web. I didn't realize this file was in an IIFE with AUI and referencing Liferay.Util directly. This is where the github PR UI can sometimes be a pain at not seeing the full context.

Yeah in that case, this is fine how you have it.

Feel free to merge

Reply:
nvm, we don't merge... i forwarded

Reply:
👍